### ISSUE TITLE
Version request

### ISSUE THREAD

**Issue by Baominchao** [pierluigiferrari/ssd_keras#309]
Please help me, every time I run the ssd300_training.ipynb I will get the following error:
Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[{{node conv1_1/convolution}}]]
	 [[{{node loss/add_34}}]]
My keras=2.2.4,tensorflow=1.13.1,python=3.6.9,cudatoolkit=10.0.130,cudnn=7.6.4
I am anaconda installed in win10. The above environment configuration is installed in a virtual environment of anaconda.Could you tell me the specific version? I tried a lot of versions, I don't know why there are problems.
